### PR TITLE
engine: add release-notes for 20.10.20

### PR DESCRIPTION
**Pull request by thaJeztah**

### CODE REVIEW

File: engine/release-notes/index.md [hunk 1]
```diff
@@ -15,6 +15,51 @@ for Docker Engine.
...
+  (`image`) used to pull the image, and otherwise will produce an error.
+
+
+### Builder
+
+- Updated handling of `image:tag@digest` references. Refer to the "Daemon" section
+  above for details.
+- Updated the classic Builder and updated BuildKit to [v0.8.3-31-gc0149372](https://github.com/moby/buildkit/commit/c014937225cba29cfb1d5161fd134316c0e9bdaa){:target="_blank" rel="noopener"},
+  provide a mitigation against [CVE-2022-39253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-39253){:target="_blank" rel="noopener"}
+  when using a Git repository as build context with the classic builder.
```
Comment (neersighted) [inline engine/release-notes/index.md:57]:
```suggestion
- Added a mitigation to the classic Builder and updated BuildKit to [v0.8.3-31-gc0149372](https://github.com/moby/buildkit/commit/c014937225cba29cfb1d5161fd134316c0e9bdaa){:target="_blank" rel="noopener"},
  for [CVE-2022-39253](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2022-39253){:target="_blank" rel="noopener"}
  when using the classic Builder with a Git URL as the build context.
```

**Comment by thaJeztah** [comment]
Thanks @neersighted @tonistiigi - I think I addressed all the comments (thanks!); PTAL in case I missed some

File: engine/release-notes/index.md (hunk 1 above)
Comment (thaJeztah) [reply engine/release-notes/index.md]: I dropped the "when using the classic Builder " here, as in this case it's the daemon-side, so could be for _both_

**Review by neersighted — APPROVED** [review]

**Comment by thaJeztah** [comment]
oh, LOL, need someone with write access to approve; let me see if there's someone

**Comment by thaJeztah** [comment]
There. fixed it 😉 😂

**Pull request merged**